### DISCUSSION TITLE
Discord community call: 8 June 2024

### DISCUSSION THREAD

**Discussion by BenJamesBen** [freesewing/freesewing#6830] [Support] [answered]
You are all invited to our Discord hangout.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This page will be updated after the call with notes.

### About

FreeSewing's Discord hangout is an informal hangout/chat/meeting that is open to everyone.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

### Details

Location: https://discord.freesewing.org/ (meeting room: 🔊voice chat)
Date: June 8th, 2024 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/jun-8-2024/4pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda

- TBD

**Comment by BenJamesBen** [comment, marked as answer]
### June 8, 2024 Community Call

#### Notes:
- Joost is on vacation, which is why work on accepting/merging PRs has temporarily ceased.
- GitHub Labels are now available for the newly-added FreeSewing Designs.
- Is this $800 USD university sewing course worth taking, or should I just self-study?
- How and why people learned how to sew.
- Getting over the fear of cutting fabric. Piecing is period.
- Fixing sewing machines.
- The instructions that come with patterns (including the ones on FreeSewing) are only guidelines, not hard rules.
- Velcro flys.
- Wedding dress patterns. Getting patterns in thrift stores.
- Why is the Magde messenger bag design in the FreeSewing repository but not released? We need construction instructions for this design. We should follow up with the designer to see if we can write the documentation and get it released.
- How to write and submit documentation for FreeSewing designs, particularly how to handle translation files. (Write and submit the English version. Do not provide translations in other languages.)

#### Attendees:
- Benjamin F.
- bobgeorgethe3rd
- Saber.007
- TangerineShark
- Thrunic
- Vili
- WeBuild4Life
- Wouter.vdub

**Comment by bobgeorgethe3rd** [comment]
https://github.com/freesewing/freesewing/issues/5169
^ Already got a tracker issue.
I have messaged cleg to see if they are still intersting in adding Magde
